### PR TITLE
stats: fix estimation for pseudo equal row count

### PR DESCRIPTION
**Pull request by lamxTyler**
The equal row count for primary key should be at most 1.
PTAL @coocood @winoros

### CODE REVIEW

File: plan/physical_plan_test.go [hunk 1]
```diff
@@ -172,8 +172,8 @@ func (s *testPlanSuite) TestDAGPlanBuilderSimpleCase(c *C) {
 		},
 		// Test PK in index double read.
 		{
-			sql:  "select * from t where t.c = 1 and t.a = 1 order by t.d limit 1",
-			best: "IndexLookUp(Index(t.c_d_e)[[1,1]]->Sel([eq(test.t.a, 1)])->Limit, Table(t))->Limit",
+			sql:  "select * from t where t.c = 1 and t.a > 1 order by t.d limit 1",
```
Comment (winoros) [inline plan/physical_plan_test.go]: Why change this?

Comment (lamxTyler) [reply plan/physical_plan_test.go]: Because for `a = 1`, it won't choose index double read, so I change it.

**Comment by coocood** [comment]
LGTM

**Pull request merged**